flipperdevices/Flipper-Android-App
# Migrate update to new BLE api

**Background**

Migration of all update modules into new api

**Changes**

- Added new feature for update
- Migrated every update submodules into new api

**Test plan**

- Open flipper android app and run update
- See flipper is update correctly with all it's flow (image changed, rebooted, location sub-ghz working)
- See all tests previously located is passed

### TODO

- Device reconnect after it being shutdown(we don't get success message on mobile)(the problem is on another layer. Not actually related to feature)
